content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
+```
+
+The complete Kmesh capability depends on the OS enhancement. Check whether the execution environment is in the [OS list](https://github.com/kmesh-net/kmesh/blob/main/docs/kmesh_support.md) supported by Kmesh. For other OS environments, see [Kmesh Compilation and Building](https://github.com/kmesh-net/kmesh/blob/main/docs/kmesh_compile.md).You can also try the [kmesh image in compatibility mode](https://github.com/kmesh-net/kmesh/blob/main/build/docker/README.md) in other OS environments.For information on various Kmesh images, please refer to the [detailed document](https://github.com/kmesh-net/kmesh/blob/main/build/docker/README.md).
+
+## Install Kmesh
+
+We offer several ways to install kmesh
+
+- Install from Helm
   
-- Start Kmesh
+```console
+helm install kmesh ./deploy/helm -n kmesh-system --create-namespace
+```
+
+- Install from Yaml
+  
+```console
+# cd ./deploy/yaml/
+kubectl apply -f kmesh.yaml
+kubectl apply -f clusterrole.yaml
+kubectl apply -f clusterrolebinding.yaml
+kubectl apply -f serviceaccount.yaml
+kubectl apply -f l7-envoyfilter.yaml
+```
+
+You can confirm the status of kmesh with the following command:
 
-  - Install from Helm
+```console
+kubectl get pod -n kmesh-system
+NAME          READY   STATUS    RESTARTS   AGE
+kmesh-v2frk   1/1     Running   0          18h
+```
 
-  ```sh
-  [root@ ~]# helm install kmesh ./deploy/helm -n kmesh-system --create-namespace
-  ```
+View the running status of kmesh service:
 
-  - Install from Yaml
+```console
+time="2024-04-25T13:17:40Z" level=info msg="bpf Start successful" subsys=manager
+time="2024-04-25T13:17:40Z" level=info msg="controller Start successful" subsys=manager
+time="2024-04-25T13:17:40Z" level=info msg="dump StartServer successful" subsys=manager
+time="2024-04-25T13:17:40Z" level=info msg="start write CNI config\n" subsys="cni installer"
+time="2024-04-25T13:17:40Z" level=info msg="kmesh cni use chained\n" subsys="cni installer"
+time="2024-04-25T13:17:41Z" level=info msg="Copied /usr/bin/kmesh-cni to /opt/cni/bin." subsys="cni installer"
+time="2024-04-25T13:17:41Z" level=info msg="kubeconfig either does not exist or is out of date, writing a new one" subsys="cni installer"
+time="2024-04-25T13:17:41Z" level=info msg="wrote kubeconfig file /etc/cni/net.d/kmesh-cni-kubeconfig" subsys="cni installer"
+time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-kindnet.conflist" subsys="cni installer"
+time="2024-04-25T13:17:41Z" level=info msg="command Start cni successful" subsys=manager
+```
+
+## Deploy the Sample Applications
+
+Similar to istio, kmesh can be used to manage applications in a namespace by adding a label to that namespace.
+
+```console
+# Enable Kmesh for the specified namespace
+kubectl label namespace default istio.io/dataplane-mode=Kmesh
+```
+
+Apply the following configuration to create sample applications:
+
+```console
+kubectl apply -f -<<EOF
```

Comment:
Could we put the configuration of sample applications to some place and only apply here?

They take up too much space

Reply:
Good suggestion, add a samples directory in kmesh repo